Multisig/Cross-layer/Analytics/TrackingEvent.swift

```diff
@@ -117,7 +117,7 @@ enum TrackingEvent: String, Trackable {
 
     case chooseOwner                                = "screen_owner_choose"
     case bannerImportOwnerKeySkipped                = "user_banner_owner_skip"
-    case bannerImportOwnerKeyImported               = "user_banner_owner_import"
+    case bannerImportOwnerKeyAdd                    = "user_banner_owner_add"
```

Comment:
I would not change tracking because it would invalidate previous tracking

Reply:
I mean tracking string value - the variable change is ok

Reply:
this was requested in the bug description. I will ask there one more time

Reply:
The change was requested during refinement meeting